docs/components/plugins/AlgoliaSearch.js

```diff
@@ -110,7 +103,8 @@ class AlgoliaSearch extends React.Component {
         return hits;
       },
       algoliaOptions: {
-        facetFilters: ignoredVersionList,
+        // include pages without version (guides/get-started) OR exact version (api-reference)
+        facetFilters: [['version:none', `version:${currentVersion}`]],
```

Comment:
I'm not sure why we don't use `latest` as version too, I don't think that will cause issues. And I think we could remove the version rewrite in the handling too 😄

Reply:
I think we ideally don't want to have `/v37.0.0` and `/latest` both indexed

Reply:
We can do that with the meta tags 😄